week2/2-website/css/style.css

```diff
@@ -5,6 +5,194 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+    background-color: white;
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+
+#logo {
+    display: flex;
+    justify-content: center;
+    width: 43vw;
+}
+#header-img {
+    width: 40px;
+    height: 40px;  
+}
+nav {
+    width: 57vw;
+    display: flex;
+    justify-content: flex-start;
+}
+nav ul li {
+    display: inline-block; 
+    padding: 10px 15px 10px 15px;
+    font-weight: 500;
+    
+}
+nav ul li:first-of-type {
+    font-weight: bold;
+}
+a {
+    text-decoration: none;
+    color: #888e99;
+}
+#imgcont {
+    margin-top: 0px;
+    background-image: url("/img/first-background.jpg");
+
+    background-size: cover;
+    color: ghostwhite;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-height: 700px;  
+    }
+h1 {
+    padding-bottom: 0;
+    color: ghostwhite;
+    font-size: 3.125rem;
+    font-weight: 300;
+    margin-bottom: 0;
+}
+p {
+    margin: 10px;
+    font-size: 1.5rem;
+    font-weight: 300;
+}
+#button {
+    text-align: center;
+    background-color: rgb(241, 87, 41);
+    font-size: 1.125rem;
+    width: 100px;
+    border-radius: 5px;
+    padding: 15px 25px;
+}
+#button a {
+    color: white;
+    font-size: 1.5;
+    font-weight: 400;
+}
+#button:hover {
+    background-color: rgb(218, 68, 23);
+}
+
+#secondSection {
```

Comment:
Try naming things a little more descriptive as you may want the same styles for similar sections later on.
Even something like `section-light-bg` can make it a little more reusable :)